src/Controls/tests/Xaml.UnitTests/Issues/Bz19560.xaml.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using NUnit.Framework;
+
+namespace Microsoft.Maui.Controls.Xaml.UnitTests
+{
+	public partial class Bz19560 : ContentPage
+	{
+		public Bz19560()
```

Comment:
BZ prefix refers to bugzilla issues. for issues filed on GH within the maui repo, we tend to use Maui19560